lib/auth.js

```
@@ -37,8 +37,16 @@ class Auth {
         }, (err, res) => {
             if (err) return cb(err);
 
-            if (res.statusCode === 404) return cb(new Error('404: Could not obtain auth list'));
-            if (res.statusCode !== 200) return cb(new Error(res.body));
+            if (res.statusCode === 404) {
+                const error = new Error('404: Could not obtain auth list');
+                error.status = res.statusCode;
```

Need to return the error here.